Makefile

```diff
@@ -24,8 +24,21 @@
 
 PKG          := ./pkg/...## Which package to run tests against, e.g. "./pkg/storage".
 TAGS         :=
-TESTS        :=.## Tests to run for use with `make test`.
-BENCHES      :=## Benchmarks to run for use with `make bench`.
+
+ifeq ($(MAKECMDGOALS),bench)
```

Comment:
This won't catch cases like `make build bench`. Can you use `ifneq ($(findstring bench,$(MAKECMDGOALS))),)` instead?

Reply:
Done